src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -319,6 +339,10 @@ class GridEditable<
       return `${COL_WIDTH_MINIMUM}px`;
     });
 
+    // The last column has no resizer and should always be a flexible column
+    // to prevent underflows.
+    widths[widths.length - 1] = `minmax(${COL_WIDTH_MINIMUM}px, auto)`;
```

Comment:
There's always a column on the table, but imo it'll be a good idea to assert that `widths.length - 1 >= 0` to stop quirky situations from breaking the world.

Reply:
People don't realize the world saving work being done here.